public/store/css/main.css

```diff
@@ -726,10 +726,9 @@ SEARCH PAGE
 */
 
 .search {
-	padding-top: 2rem;
-	background-color: #FFF;
-	width: 80%;
-	margin-left: 10%;
+    background-color: #FFF;
```

Comment:
tabbyyyyy

Reply:
Whattt. I think this is because I restored the code from a stash and the existing tabs space were lumpy.

Reply:
ah probably 😂 my text editor is awful, so might have been me.. 👻